Documentation/ApiOverview/Database/Index.rst

```diff
@@ -1,29 +1,46 @@
 :navigation-title: Database
-.. include:: /Includes.rst.txt
-.. index:: ! Database
-.. _database:
+..  include:: /Includes.rst.txt
+..  index:: ! Database
+..  _database:
 
 ========================
 Database (Doctrine DBAL)
 ========================
 
-**Contents:**
+This chapter describes accessing the database on the level of the Doctrine
+database abstraction layer (DBAL).
 
-.. toctree::
-   :titlesonly:
+The Doctrine Database Abstraction Layer (DBAL) in TYPO3 provides developers
```

Comment:
Sounds a bit like "we (TYPO3) have build DBAL". I prefer something like "With help of DBAL..." or "DBAL is a PHP community driven ... which is used in TYPO3 since ..." or maybe just a simple link to the project is missing to clearify that.

Reply:
I linked the project, good point